### PR TITLE
Cephs3-test: CopyObject testcase failing with error when calling puto…

### PR DESCRIPTION
**Pull request by shalakadharap**
…bjectacl operation

### CODE REVIEW

**Comment by nileshgovande** [comment]
@shalakadharap Pls contact Shazia and make sure that the test is enabled back in ceph and splunk job
She will point you to the entry needs to be added.

**Comment by shalakadharap** [comment]
image

**Comment by shalakadharap** [comment]
Difference between original and modified/new xsd

**Comment by nileshgovande** [comment]
Shalaka Dharap
1)Work with Shazia to enable the script back (the script is in s3 repo itself) in the same PR
2)Make sure the ceph and splunk job passes on ur branch

File: auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java [hunk 1]
```diff
@@ -58,7 +58,7 @@
     public
      static final String DEFAULT_ACL_XML = "/defaultAclTemplate.xml";
     public
-     static final String XSD_PATH = "/AmazonS3.xsd";
+     static final String XSD_PATH = "/AmazonS3_V2.xsd";
```
Comment (bkirunge7) [inline auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java:61]: Add a ST test case set object acl with reordered access control list

**Review by bkirunge7 — APPROVED** [review]

**Comment by sali619** [comment]
> Shalaka Dharap
> 1)Work with Shazia to enable the script back (the script is in s3 repo itself) in the same PR
> 2)Make sure the ceph and splunk job passes on ur branch

**Comment by sali619** [comment]
By Mistake it got closed,Reopening it again.

File: auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java (hunk 1 above)
Comment (shalakadharap) [reply auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java:61]: Done

File: auth/resources/AmazonS3_V2.xsd [hunk 2]
```diff
@@ -0,0 +1,701 @@
...
+    <xsd:restriction base="xsd:string">
+      <xsd:enumeration value="READ"/>
+      <xsd:enumeration value="WRITE"/>
+      <xsd:enumeration value="READ_ACP"/>
+      <xsd:enumeration value="WRITE_ACP"/>
+      <xsd:enumeration value="FULL_CONTROL"/>
+    </xsd:restriction>
+  </xsd:simpleType>
+
+  <xsd:simpleType name="StorageClass">
```
Comment (sachitanands) [inline auth/resources/AmazonS3_V2.xsd:224]: is this "StorageClass" supported in our case ?

File: auth/resources/AmazonS3_V2.xsd [hunk 3]
```diff
@@ -0,0 +1,701 @@
...
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="CreateBucketConfiguration">
+    <xsd:sequence>
+      <xsd:element name="LocationConstraint" type="tns:LocationConstraint"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="LocationConstraint">
```
Comment (sachitanands) [inline auth/resources/AmazonS3_V2.xsd:252]: same as above

File: auth/resources/AmazonS3_V2.xsd [hunk 4]
```diff
@@ -0,0 +1,701 @@
...
+  </xsd:simpleType>
+
+  <xsd:simpleType name="VersioningStatus">
+    <xsd:restriction base="xsd:string">
+      <xsd:enumeration value="Enabled"/>
+      <xsd:enumeration value="Suspended"/>
+    </xsd:restriction>
+  </xsd:simpleType>
+
+  <xsd:complexType name="NotificationConfiguration">
```
Comment (sachitanands) [inline auth/resources/AmazonS3_V2.xsd:688]: same as above

File: auth/resources/AmazonS3_V2.xsd (hunk 2 above)
Comment (shalakadharap) [reply auth/resources/AmazonS3_V2.xsd:224]: Existing. Only difference is pasted below.

File: auth/resources/AmazonS3_V2.xsd (hunk 3 above)
Comment (shalakadharap) [reply auth/resources/AmazonS3_V2.xsd:252]: Existing. Only difference is pasted below.

File: auth/resources/AmazonS3_V2.xsd (hunk 4 above)
Comment (shalakadharap) [reply auth/resources/AmazonS3_V2.xsd:688]: Existing. Only difference is pasted below.

**Review by sachitanands — APPROVED** [review]

**Comment by shalakadharap** [comment]
Ceph tests passed
Verified with ceph job below-
http://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/S3Server/job/S3-Compatibility-Test-Ceph/149/

**Pull request merged**